dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsReaderTest.java

```diff
@@ -143,6 +147,13 @@ public void testQuery() throws Exception {
         return list;
     }
 
+    private void checkSnpOrientation(List<SubSnpCoreFields> readSnps, Long snpId, Orientation snpOrientation, Orientation contigOrientation) {
+        Optional<SubSnpCoreFields> snp = readSnps.stream().filter(s -> s.getRsId().equals(snpId)).findAny();
```

Comment:
for now it's fine, but in the code that adds alleles, this method will change because several alternate alleles can be mapped to the same rsId